articles/qdk-relnotes.md

```diff
@@ -19,6 +19,13 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update-qdk).
 
+## Version X.XX.XXXX.XXXXXX
+
+*Release date: February 22th, 2021*
+
+- Improved IQ# debug user experience, to help developers better understand the inner workings of their quantum applications, and more easily identify logic errors. Created visualizations of a particular execution path of a quantum application, as well as the state at a point during the program execution. For both execution path and basis state visualizations, a horizontal scrollbar is added to scroll through the additional content. 
```

Comment:
Only part of the IQ# debug improvement work is in this release, the rest will come next month. Maybe something like this:
 `Improved IQ# debug user experience by adding a horizontal scrollbar to scroll both execution path and basis state visualizations`?